roles/certificates/templates/openssl.cnf.j2

```diff
@@ -28,44 +28,32 @@ unique_subject  = no
 copy_extensions = copy
 
 [ policy_match ]
-countryName            = match
-stateOrProvinceName    = match
-organizationName       = match
-organizationalUnitName = optional
-commonName             = supplied
-emailAddress           = optional
+countryName             = optional
+stateOrProvinceName     = optional
+localityName            = optional
+organizationName        = optional
+organizationalUnitName  = optional
+commonName              = supplied
+emailAddress            = optional
 
 [ req ]
-default_bits        = 2048
+default_bits        = 4096
 default_keyfile     = privkey.pem
-distinguished_name  = req_distinguished_name
 req_extensions      = v3_req
 prompt              = no
 
-[ req_distinguished_name ]
-countryName            = US
-stateOrProvinceName    = NC
-localityName           = Raleigh
-organizationName       = Foreman
-commonName             = {{ ansible_fqdn }}
-
 [ v3_ca ]
 basicConstraints        = CA:TRUE
 subjectKeyIdentifier    = hash
 authorityKeyIdentifier  = keyid:always,issuer:always
 nsCertType              = sslCA
 keyUsage                = cRLSign, keyCertSign
 extendedKeyUsage        = serverAuth, clientAuth
-subjectAltName          = @alt_names
 
 [ v3_req ]
 basicConstraints     = CA:FALSE
 subjectKeyIdentifier = hash
 extendedKeyUsage     = serverAuth, clientAuth
-subjectAltName       = @alt_names
```

Comment:
Don't we need a SAN on the certs?

Reply:
Added back, via CLI since it needs to be dynamic.